account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -395,19 +396,36 @@ def _prepare_line_vals_nline(self, partner, vals, parsed_inv, import_config):
                 )
             if line.get("name"):
                 il_vals["name"] = line["name"]
+            if line.get("line_note"):
+                il_vals = {
```

Comment:
Can't we avoid all these repetitions of all the values?
Besides, this method is already very long, it would be nice to do some splitting...